src/main/java/com/flightstats/hub/filter/DataDogRequestFilter.java

```diff
@@ -52,28 +54,44 @@ public void filter(ContainerRequestContext request, ContainerResponseContext res
     public static void finalStats() {
         try {
             DataDogState dataDogState = threadLocal.get();
-            if (null == dataDogState) {
-                return;
-            }
+            if (null == dataDogState) return;
```

Comment:
In general, we've been using the verbose braces format.  If we decide to switch to no braces, we should do that to the project.